docs/devGuide/development/migratingToTypeScript.md

```diff
@@ -95,6 +95,31 @@ We have decided on structuring the commits as two commits: a "Rename" commit and
 
 You are now ready to create a pull request for the changes to the repository.
 
+### (Optional) Step 4: Changing the commit
+
+While refactoring you may encounter scenarios where you might have to amend your code after a commit. There are 2 common scenarios which we will cover. 
+
+##### Scenario 1 - you have 2 commits and want to add on to the second commit: 
```

Comment:
nit: can we exclude these level-5 headers from the page nav? The headers are quite long and may clutter up the page nav.
Or maybe we can put it in a panel since this section is optional?

Reply:
Good idea, ill put this into a panel!